### PR TITLE
Catwalk Flooring

### PR DESCRIPTION
**Pull request by tralezab**
## About The Pull Request

looks like this:
image

## Why It's Good For The Game

Would look GREAT in engineering ;)

## Changelog
:cl:
add: catwalk flooring!
/:cl:

### CODE REVIEW

**Comment by dondonpow** [comment]
i will pay you $20 to never say "stan" again.

**Comment by CRITAWAKETS** [comment]
Only one color of catwalk flooring and its the green one? This is basically never gonna get used, just saying.

**Comment by tralezab** [comment]
thats like a dark grey! we have tons of tiles that use that :)

**Comment by Greniza** [comment]
I agree with Critawakets. You should add color variants for the edges so that it matches more tiles than the single least-popular tile variant. I'd also recommend adding turf decals for it, as well as perhaps maybe directional variants to join multiple tiles together. Overall, though, good concept!

**Comment by tralezab** [comment]
i'll give you a MAYBE. i may add more variants, i like this turf

**Comment by JJRcop** [comment]
Thia doea not include a crafting recipe

**Comment by MMMiracles** [comment]
i like this turf but does it also come with the mid-flight rocket

**Comment by Technoturnovers** [comment]
this looks cool as hell, and is also really useful if you want to have some kind of flooring but still want to see the pipes and wires

**Comment by MrMelbert** [comment]
Kinda wish our already existing catwalk structures could use those footstep sounds.

**Comment by Zytolg** [comment]
I needed this a year ago <3

**Comment by JJRcop** [comment]
I think it's important to note that this PR does not allow you to attain these tiles, only adds them to the code

File: code/game/turfs/open/floor/catwalk_plating.dm [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+/turf/open/floor/plating/catwalk_floor/screwdriver_act(mob/living/user, obj/item/tool)
+	. = ..()
+	covered = !covered
+	user.balloon_alert(user, "[!covered ? "cover removed" : "cover added"]")
+	update_icon(UPDATE_OVERLAYS)
+
+/turf/open/floor/plating/catwalk_floor/pry_tile(obj/item/crowbar, mob/user, silent)
+	if(covered)
+		user.balloon_alert(user, "remove cover first!")
```
Comment (Kylerace) [inline code/game/turfs/open/floor/catwalk_plating.dm:40]:
```suggestion
		user.balloon_alert(user, "remove the cover first!")
```

**Review by Kylerace — CHANGES_REQUESTED** [review]

File: code/game/turfs/open/floor/catwalk_plating.dm (hunk 1 above)
Comment (Mothblocks) [reply code/game/turfs/open/floor/catwalk_plating.dm:40]: akshually balloon alerts are intentionally kept extremely short due to their ephemeral nature

**Comment by Twaticus** [comment]
grey/black/white tile versions? 😳

**Review by Krysonism — APPROVED** [review]
Looks good.

**Comment by JJRcop** [comment]
This is cool, but how will you actually see this in game? I don't see anything in this PR that does that

**Comment by JJRcop** [comment]
This is cool, but how will you actually see this in game? I don't see anything in this PR that does that.

**Pull request merged**